src/Lean/Meta/WHNF.lean

```diff
@@ -314,7 +314,7 @@ end
   | .lam ..        => return e
   | .sort ..       => return e
   | .lit ..        => return e
-  | .bvar ..       => unreachable!
+  | .bvar ..       => panic! "whnf: loose bvar in expression"
```

Comment:
Note that including "whnf: " here might be seen as being redundant. This message will display as
```
PANIC at Lean.Meta.whnfEasyCases Lean.Meta.WHNF:317:22: whnf: loose bvar in expression
```
I'm not sure that's a bad thing, but I just wanted to point it out.